bibtex-actions.el

```diff
@@ -84,6 +84,15 @@ may be indicated with the same icon but a different face."
   :group 'bibtex-actions
   :type 'string)
 
+(defcustom bibtex-actions-force-refresh-hook nil
+  "Hook run when user forces a (re-) building of the candidates cache.
+This hook is only called when the user explicitly requests that
+the cache has to be rebuilt. It is intended for 'heavy'
+operations such as a complete export of a Zotero library to a
```

Comment:
Aside from addressing the warnings, while I use Zotero myself, could we have a more generic example?

Not a big deal if not ....